backend/db/refresh.py

```diff
@@ -37,19 +37,27 @@ def refresh_db(
     this is set to True."""
     print('INFO: Starting database refresh.', flush=True)  # flush: for gh action
     t0, t0_str = datetime.now(), current_datetime()
-    with get_db_connection(local=use_local_database) as con:
-        last_refresh = last_refresh_timestamp(con)
-        if since and dp.parse(since) > dp.parse(last_refresh) and not force_non_contiguity:
-            raise ValueError(SINCE_ERR)
-        since = since if since else last_refresh
+    update_db_status_var('last_refreshed_request', t0_str)
+    update_db_status_var('refresh_status', 'active')
+    try:
+        with get_db_connection(local=use_local_database) as con:
+            last_refresh = last_refresh_timestamp(con)
+            if since and dp.parse(since) > dp.parse(last_refresh) and not force_non_contiguity:
+                raise ValueError(SINCE_ERR)
+            since = since if since else last_refresh
 
-        # Refresh DB
+            # Refresh DB
         csets_and_members_enclave_to_db(con, schema, since)
 
-    update_db_status_var('last_refreshed_DB', t0_str)
-    counts_update('DB refresh.', schema, use_local_database)
-    print(f'INFO: Database refresh complete in {(datetime.now() - t0).seconds} seconds.')
+        counts_update('DB refresh.', schema, use_local_database)
+        update_db_status_var('last_refreshed_success', current_datetime())
```

Comment:
@hugwuoke From what I can see everything looks exactly as I'd hoped! Thanks. I'm gonna give it a test run.